Wikipedia/Code/ProfileCoordinator.swift

```diff
@@ -178,6 +183,42 @@ final class ProfileCoordinator: NSObject, Coordinator, ProfileCoordinatorDelegat
         yirCoordinator.start()
     }
     
+    private func presentLoginPrompt() {
```

Comment:
@mazevedofs Are you okay with us changing the coding pattern like this? We're having issues with layering the navigation and views

Reply:
Do you mean it's not working as-is (calling the coordinator here?). Otherwise, I'm good with it. It uses the Login Coordinator to navigate to login, so we're still keeping the Coordinator pattern, which is my biggest concern.